samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -117,7 +117,18 @@ The RADIO can control up to 8 GPIO pins for the purpose of controlling the exter
 
 The antenna is chosen by writing consecutive numbers to the SWITCHPATTERN register.
 This means that the antenna GPIO pins act like 8-bit registers.
-In other words, for the first antena, antenna pin 1 is active, for the second antenna, pin 2 is active, for the third antenna, pins 1 and 2 are active, and so on.
+In other words, for the first antenna, antenna pin 1 is active, for the second antenna, pin 2 is active, for the third antenna, pins 1 and 2 are active, and so on.
+
+nRF21540 Front-End-Module
+=========================
+
+The DTM sample supports nRF21540 Front-End-Module.
+
+This allows an increase of the transmitted power through a Power Amplifier (PA) or an increase of the sensitivity through a Low-Noise Amplifier (LNA).
+Any increase in power and sensitivity results in an increased communication range.
+The nRF21540 transmitted power gain and an activation delay can be configured using vendor specific commands `Vendor-Specific packet payload`_.
+
+See the `nRF21540 Front-End-Module Product Specification`_ for more details
 
 Vendor-Specific packet payload
 ==============================
```

Comment:
No need to capitalize "Specific" -> Vendor-specific

Reply:
Fixed